openedx_events/learning/data.py

```diff
@@ -296,3 +296,17 @@ class ProgramCertificateData:
     status = attr.ib(type=str)
     url = attr.ib(type=str)
     certificate_available_date = attr.ib(type=datetime, default=None)
+
+
+@attr.s(frozen=True)
+class CourseStaffData:
+    """
+    Attributes defined for the Open edX Authentication function to Add Users to a role.
```

Comment:
can we be less specific here? It could say something like: Attributes used to represent course staff. or something like it but less specific on how it'd be used.

Reply:
I've changed the name to `ManageStudentsPermissionsData` to reflect planned changes to roles/permissions.